packages/pds/src/auth.ts

```diff
@@ -26,36 +42,56 @@ export class ServerAuth {
     this.verifier = new auth.Verifier({ didResolver: opts.didResolver })
   }
 
-  createToken(did: string): string {
-    return jwt.sign(
-      {
-        sub: did,
-      },
-      this._secret,
-    )
+  createAccessToken(did: string, expiresIn?: string | number) {
+    const payload = {
+      scope: AuthScopes.Access,
+      sub: did,
+    }
+    return {
+      payload: payload as AuthToken, // exp set by sign()
+      jwt: jwt.sign(payload, this._secret, {
+        expiresIn: expiresIn ?? '15mins',
+        mutatePayload: true,
+      }),
+    }
   }
 
-  getUserDid(req: express.Request): string | null {
-    const header = req.headers.authorization || ''
-    if (!header.startsWith(BEARER)) return null
-    const token = header.slice(BEARER.length)
-    const payload = jwt.verify(token, this._secret)
+  createRefreshToken(did: string, expiresIn?: string | number) {
+    const payload = {
+      scope: AuthScopes.Refresh,
+      sub: did,
+      jti: uint8arrays.toString(crypto.randomBytes(32), 'base64'),
```

Comment:
@dholms do you feel this is appropriate for generating a unique id?  Typically I would just reach for a uuid, but I saw they aren't used anywhere else, so I reached for this instead.

Reply:
yup this seems fine to me 👍 
32 random bytes is enough to not worry about collisions